packages/xdl/src/Android.ts

```diff
@@ -235,9 +321,23 @@ async function _openUrlAsync(url: string) {
   return output;
 }
 
+async function attemptToStartEmulatorOrAssertAsync() {
+  if (!(await _isDeviceAttachedAsync())) {
+    // If no devices or emulators are attached we should attempt to open one.
+    if (!(await maybeStartAnyEmulatorAsync())) {
+      // TODO(Bacon): Maybe add instructions for easily creating an emulator.
```

Comment:
we can link to these instructions: https://docs.expo.io/versions/latest/workflow/android-studio-emulator/